### PR TITLE
Feature/search sbi

### PR DESCRIPTION
**Pull request by kaz-iyiola**

### CODE REVIEW

File: test/acceptance/wdio.conf.js [hunk 1]
```diff
@@ -29,6 +29,16 @@ exports.config = {
       // //  args: chromeArgs
       // }
     }
+
```
Comment (pxp420) [inline test/acceptance/wdio.conf.js:32]: is this needed as it looks commented

File: test/acceptance/page-objects/eligible-organisation.js [hunk 2]
```diff
@@ -21,11 +24,49 @@ class eligibleOrganisation extends Page {
...
+
+ 
+  async verifySbiErrorMessage() {
+    console.log(await (await this.sbiError).getText());
+    const sbiErrorText = await (await this.sbiError).getText();
+    chai.expect(sbiErrorText).to.not.be.empty;
+    console.log(sbiErrorText);
+    return sbiErrorText;   
+  }
+
```
Comment (pxp420) [inline test/acceptance/page-objects/eligible-organisation.js:55]: remove extra line here

File: test/acceptance/page-objects/eligible-organisation.js [hunk 3]
```diff
@@ -21,11 +24,49 @@ class eligibleOrganisation extends Page {
...
 
+  
+  async verifySbiNumber() {
+    console.log(await (await this.sbiNum).getText());
+    const sbiNumber = await (await this.sbiNum).getText();
+    chai.expect(sbiNumber).to.not.be.empty;
+    console.log(sbiNumber);
+    return sbiNumber;  
+  }
+
```
Comment (pxp420) [inline test/acceptance/page-objects/eligible-organisation.js:46]: remove extra line here

File: test/acceptance/page-objects/eligible-organisation.js [hunk 4]
```diff
@@ -21,11 +24,49 @@ class eligibleOrganisation extends Page {
...
+  get sbiNum () { return $('//td[2]') }   
+  get sbiError () { return $('#error-message')}
+  
+  
   open () {
     super.open('')
     browser.pause(3000)
   }
 
+
```
Comment (pxp420) [inline test/acceptance/page-objects/eligible-organisation.js:38]: remove extra line here

File: test/acceptance/page-objects/eligible-organisation.js [hunk 5]
```diff
@@ -21,11 +24,49 @@ class eligibleOrganisation extends Page {
   get sbi18 () { return $('//tr[18]/td[4]/a') }
   get sbi19 () { return $('//tr[19]/td[4]/a') }
 
+  //searh sbi number element
+  get sbiNumberField () { return $('#user-search') }
+  get sbiNum () { return $('//td[2]') }   
+  get sbiError () { return $('#error-message')}
+  
+
```
Comment (pxp420) [inline test/acceptance/page-objects/eligible-organisation.js:32]: remove extra line here

**Review by pxp420 — COMMENTED** [review]
Just some tidying to do but apart from that all good!

File: test/acceptance/wdio.conf.js (hunk 1 above)
Comment (kaz-iyiola) [reply test/acceptance/wdio.conf.js:32]: The commented lines are removed

File: test/acceptance/page-objects/eligible-organisation.js (hunk 2 above)
Comment (kaz-iyiola) [reply test/acceptance/page-objects/eligible-organisation.js:55]: Done

File: test/acceptance/page-objects/eligible-organisation.js (hunk 3 above)
Comment (kaz-iyiola) [reply test/acceptance/page-objects/eligible-organisation.js:46]: Done

File: test/acceptance/page-objects/eligible-organisation.js (hunk 4 above)
Comment (kaz-iyiola) [reply test/acceptance/page-objects/eligible-organisation.js:38]: Done

File: test/acceptance/page-objects/eligible-organisation.js (hunk 5 above)
Comment (kaz-iyiola) [reply test/acceptance/page-objects/eligible-organisation.js:32]: Done

**Review by pxp420 — APPROVED** [review]
LGTM

**Review by pxp420 — APPROVED** [review]

**Pull request merged**